PETSc example: Areasphere (#439)
CEED/libCEED: examples/README.md

The nodal points, *p*, are Gauss-Legendre-Lobatto, and the quadrature points, *q* are Gauss-Legendre-Lobatto. The nodal points and quadrature points are collocated.

## PETSc+libCEED Navier-Stokes Solver

The Navier-Stokes problem solves the compressible Navier-Stokes equations using an explicit time integration. A more detailed description of the problem formulation
can be found in the [`navier-stokes`](./navierstokes) folder.

## PETSc+libCEED Surface Area Examples

These examples use the mass operator to compute the surface area of a cube or a discrete cubed-sphere, using PETSc.

These examples show in particular the constructions of geometric factors to handle problems in which the elements topological dimension is different from the

> geometrical dimension and for which the coordinate transformation Jacobian from 3D physical space, to 2D local and reference spaces is a non-square matrix.

Reviewer: ```suggestion
geometrical dimension and for which the coordinate transformation Jacobian from 2D reference space to a manifold embedding in 3D physical space is a non-square matrix.
```

Author: Thanks!

./navierstokes -ceed /cpu/self
./navierstokes -ceed /gpu/occa
cd ..

cd petsc
make
./area -problem cube -ceed /cpu/self -petscspace_degree 3
./area -problem cube -ceed /gpu/occa -petscspace_degree 3

> ./area -problem sphere /cpu/self -petscspace_degree 3 -dm_refine 2

Reviewer: ```suggestion
./area -problem sphere -ceed /cpu/self -petscspace_degree 3 -dm_refine 2
```

Author: Duh! Thanks!

> ./area -problem sphere /gpu/occa -petscspace_degree 3 -dm_refine 2

Reviewer: ```suggestion
./area -problem sphere -ceed /gpu/occa -petscspace_degree 3 -dm_refine 2
```
